test/utils/envtest/environment.go

```diff
@@ -84,11 +82,16 @@ func (e *Environment) Stop() error {
 
 // InitState creates a new isolated environment with its own namespace.
 func (e *Environment) InitState(ctx context.Context) (*State, error) {
+	return InitStateWithNamespace(ctx, e.Client)
+}
+
+// InitState creates a new isolated environment with its own namespace.
+func InitStateWithNamespace(ctx context.Context, c client.Client) (*State, error) {
 	state := NewState()
 	// create a new testing namespace
 	ns := &corev1.Namespace{}
 	ns.GenerateName = "unit-tests-"
```

Comment:
Not sure, if it is used in a different context, shouldn't this be `integration-tests-` ?

Reply:
its also used in unit tests. So will just use `tests-` as prefix